docs/generated/packages/workspace/documents/nx-nodejs-typescript-version-matrix.md

```diff
@@ -13,8 +13,9 @@ and the version of NodeJS that we tested it against.
 
 | Nx Version      | Node Version     | Typescript Version |
 | --------------- | ---------------- | ------------------ |
-| 20.x (latest)   | 22.x, 20.x       | ~5.4.2             |
-| 19.x (previous) | 22.x, 20.x, 18.x | ~5.4.2             |
+| 21.x (latest)   | 22.x, ^20.19.0   | ~5.4.2             |
+| 20.x (previous) | 22.x, 20.x, 18.x | ~5.4.2             |
```

Comment:
We need to fix the currently live site by cherry-picking this commit onto the `website` branch so it needs to make sense for right now:

```suggestion
| 21.x (next)   | 22.x, ^20.19.0   | ~5.4.2             |
| 20.x (latest) | 22.x, 20.x, 18.x | ~5.4.2             |
```